### PR TITLE
Update banking with facebooks

### PR DESCRIPTION
**Pull request by stephengroat**
diff looks bad, but I didn't chance that much

### CODE REVIEW

**Comment by fpigerre** [comment]
@stephengroat It looks like this branch has merge conflicts. Do you think you would be able to fix them before merging?

File: _data/banking.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ websites:
     - name: Actors Federal Credit Union
       url: http://www.actorsfcu.com/
```
Comment (Jawshy) [inline _data/banking.yml]: Redirects to HTTPS. Please update.

File: _data/banking.yml [hunk 2]
```diff
@@ -49,6 +54,7 @@ websites:
       img: mediolanum.png
       tfa: No
       twitter: BancaMediolanum
+      facebook: BancaMediolanum
```
Comment (Jawshy) [inline _data/banking.yml:57]: ~~Should the `lang: it` (Italian) field be added?~~

File: _data/banking.yml [hunk 3]
```diff
@@ -145,12 +156,14 @@ websites:
     - name: CIBC
       url: http://www.cibc.ca
```
Comment (Jawshy) [inline _data/banking.yml]: Now supporting HTTPS. Please update.

File: _data/banking.yml [hunk 4]
```diff
@@ -145,12 +156,14 @@ websites:
...
       twitter: cibc
+      faceook: CIBC
       img: cibc.png
       tfa: No
 
     - name: Citibank
       url: https://online.citibank.com/US/Welcome.c
```
Comment (Jawshy) [inline _data/banking.yml]: Please use `https://online.citi.com/`.

File: _data/banking.yml [hunk 5]
```diff
@@ -165,12 +178,14 @@ websites:
     - name: Citizens Bank
       url: https://www.citizensbank.com
       twitter: citizensbank
+      facebook: citizensbank
       img: citizensbank.png
       tfa: No
 
     - name: comdirect
```
Comment (Jawshy) [inline _data/banking.yml:186]: ~~Could the `lang: de` field be added for `comdirect`?~~

File: _data/banking.yml [hunk 6]
```diff
@@ -281,12 +299,14 @@ websites:
     - name: GE Capital Bank
       url: https://www.gecapitalbank.com
       twitter: GECapital
+      facebook: GECapitalOfficial
       img: GECapitalBank.png
       tfa: No
 
     - name: Halifax
       url: http://www.halifax.co.uk
```
Comment (Jawshy) [inline _data/banking.yml]: HTTPS is available. Please update.

File: _data/banking.yml [hunk 7]
```diff
@@ -329,12 +350,14 @@ websites:
     - name: LendingClub
       url: https://www.lendingclub.com/
       twitter: lendingclub
+      faceook: LendingClubTeam
       img: lendingclub.png
       tfa: No
 
     - name: Lloyds Bank
       url: http://www.lloydsbank.com/
```
Comment (Jawshy) [inline _data/banking.yml]: HTTPS works. Please update.

File: _data/banking.yml [hunk 8]
```diff
@@ -343,23 +366,27 @@ websites:
       img: mtbank.png
       tfa: No
       twitter: MandT_Help
+      faceook: MandTBank
 
     - name: MBNA Canada
       url: http://www.mbna.ca
```
Comment (Jawshy) [inline _data/banking.yml]: Redirects to `https://www.mbna.ca/`. Please update.

File: _data/banking.yml [hunk 9]
```diff
@@ -428,18 +456,21 @@ websites:
...
       tfa: No
 
     - name: PNC Bank
       url: https://www.pnc.com/
       twitter: pncnews
+      facebook: pncbank
       img: pncbank.png
       tfa: No
 
     - name: Postbank
```
Comment (Jawshy) [inline _data/banking.yml:472]: ~~Perhaps change the `url` to `https://www.postbank.de/` and add `lang: de` for Postbank?~~

File: _data/banking.yml [hunk 10]
```diff
@@ -545,6 +586,7 @@ websites:
     - name: Tangerine
       url: http://www.tangerine.ca/
```
Comment (Jawshy) [inline _data/banking.yml]: Now supports HTTPS. Please update.

File: _data/banking.yml [hunk 11]
```diff
@@ -641,18 +692,21 @@ websites:
...
 
     - name: Zag Bank
       url: https://www.zagbank.ca/
       twitter: zagbank
+      facebook: zagbank
       img: zagbank.png
       tfa: No
 
     - name: ZenBanx
       url: http://zenbanx.ca/
```
Comment (Jawshy) [inline _data/banking.yml]: Redirects to HTTPS. Please update.

**Review by Jawshy — COMMENTED** [review]
Facebook links `looks-good`. `url` fields could be updated and `lang` fields added for certain sites.

**Comment by stephengroat** [comment]
@Jawshy you know you can update these too, right?

**Comment by Jawshy** [comment]
Updated.

File: _data/banking.yml [hunk 12]
```diff
@@ -143,14 +155,16 @@ websites:
       doc: /notes/chase/
 
     - name: CIBC
-      url: http://www.cibc.ca
+      url: https://www.cibc.ca
       twitter: cibc
+      faceook: CIBC
```
Comment (jamcat22) [inline _data/banking.yml]: Line should be changed to `facebook: CIBC`.

File: _data/banking.yml [hunk 13]
```diff
@@ -309,6 +331,7 @@ websites:
     - name: Huntington National Bank
       url: https://www.huntington.com
       twitter: Huntington_Bank
+      faceook: HuntingtonBank
```
Comment (jamcat22) [inline _data/banking.yml]: Line should be changed to `facebook: HuntingtonBank`.

File: _data/banking.yml [hunk 14]
```diff
@@ -329,12 +352,14 @@ websites:
     - name: LendingClub
       url: https://www.lendingclub.com/
       twitter: lendingclub
+      faceook: LendingClubTeam
```
Comment (jamcat22) [inline _data/banking.yml]: Line should be changed to `facebook: LendingClubTeam`.

File: _data/banking.yml [hunk 15]
```diff
@@ -329,12 +352,14 @@ websites:
...
       img: lendingclub.png
       tfa: No
 
     - name: Lloyds Bank
-      url: http://www.lloydsbank.com/
+      url: https://www.lloydsbank.com/
       twitter: AskLloydsBank
+      faceook: lloydsbank
```
Comment (jamcat22) [inline _data/banking.yml]: Line should be changed to `facebook: lloydsbank`.

File: _data/banking.yml [hunk 16]
```diff
@@ -343,23 +368,27 @@ websites:
       img: mtbank.png
       tfa: No
       twitter: MandT_Help
+      faceook: MandTBank
```
Comment (jamcat22) [inline _data/banking.yml]: Line should be changed to `facebook: MandTBank`.

File: _data/banking.yml [hunk 17]
```diff
@@ -343,23 +368,27 @@ websites:
...
       img: mbnacanada.png
       tfa: No
       twitter: mbna_canada
+      facebook: MBNACanada
 
     - name: Meridian Credit Union
       url: https://www.meridiancu.ca
       img: meridian.png
       twitter: meridiancu
+      faceook: MeridianCreditUnion
```
Comment (jamcat22) [inline _data/banking.yml]: Line should be changed to `facebook: MeridianCreditUnion`.

File: _data/banking.yml [hunk 18]
```diff
@@ -381,6 +410,7 @@ websites:
       img: navyfederal.png
       tfa: No
       twitter: NavyFederal
+      faceook: NavyFederal
```
Comment (jamcat22) [inline _data/banking.yml]: Line should be changed to `facebook: NavyFederal`.

**Review by jamcat22 — CHANGES_REQUESTED** [review]
Multiple lines are misspelled as `faceook:` instead of `facebook:`.

**Review by jamcat22 — APPROVED** [review]

**Pull request merged**